site/index.md

```diff
@@ -26,11 +26,8 @@ glossary
 ```
 ## About
 
-In mid-2020, the NumPy Survey Team in partnership with students and faculty
-from a Masters course in Survey Methodology jointly hosted by the
-University of Michigan and the Unverisity of Maryland conducted the first
-official NumPy Community Survey.
-We invited all NumPy stakeholders to share their experiences with NumPy, and
-to voice their thoughts about the future of the project.
-The results from this inaugural survey can be found
-{doc}`here <content/2020/index>`.
+Last July, the NumPy survey team conducted an annual NumPy community survey. 
+We invited all NumPy stakeholders to share their experiences with NumPy, 
+and to voice their thoughts about the future of the project. 
+The results from this survey can be found
+{doc}`here <content/2021/index>`.
```

Comment:
Do we want to add a quick blurb here about the genesis of the community survey? It might be nice to provide context for when/how the survey started. The text doesn't need to be nearly as detailed as it currently is, maybe just a quick sentence with a link to the 2020 acknowldegements, something like:

```suggestion
{doc}`here <content/2021/index>`.

The inaugural NumPy community survey was developed in partnership with
students and faculty from the Master's course in Survey Methodology
jointly hosted by the University of Michigan and the University of Maryland; see
{doc}`here <content/2020/acknowledgements>` for further details.
```

Just an idea - no strong feelings either way!